src/core/java/app/com/linkedin/playparseq/j/PlayParSeqImpl.java

```diff
@@ -37,7 +36,7 @@
   /**
    * The field DEFAULT_TASK_NAME is the default name of ParSeq Task.
    */
-  public final static String DEFAULT_TASK_NAME = "fromPlayPromise";
+  public final static String DEFAULT_TASK_NAME = "fromCompletionStage";
```

Comment:
I think `fromJavaCompletionStage` is better.

Reply:
I don't think Java is needed to clarify things since Scala doesn't have CompletionStages. But it might be clearer if Play is in the name to indicate it's happening due to a conversion at the framework level

Reply:
Sounds good!